src/github-issues-prs.ts

```diff
@@ -417,7 +422,7 @@ export class GitHubIssuesPrsProvider implements TreeDataProvider<TreeItem> {
 				const [, owner, repo] = m;
 				let remote = remotes[`${owner}/${repo}`];
 				if (!remote) {
-					const url = `https://github.com/${owner}/${repo}.git`;
+					const url = `https://${this.host}/${owner}/${repo}.git`;
```

Comment:
Does this not need the encoding of `getURL()`? Do you have an example of when the encoding is needed?

Reply:
e.g. `Thyme &time=again` to `comment=Thyme%20&time=again`.
However, I think host name doesn't have space.
I'll remove this. Thank you.